code/modules/unit_tests/_unit_tests.dm

```diff
@@ -0,0 +1,11 @@
+//include unit test files in this module in this ifdef
+//Keep this sorted alphabetically
+
+#ifdef UNIT_TESTS
```

Comment:
Tests are only included if they are specifically compiled in. This stops people from being able to reboot the server when they shouldnt be able to

Reply:
They are needed by the `FinishTestRun()` proc, so unless I only compile that proc in when the define is set, I need the vars to always exist